src/SeqViz/Circular/Labels.tsx

```diff
@@ -320,3 +349,20 @@ export default class Labels extends React.Component {
     );
   }
 }
+
+interface LabelWithCoors {
+  label: Label;
+  lineCoor: Coor;
+  textCoor: Coor;
+  textAnchor: unknown;
+}
+
+interface GroupedLabelsWithCoors {
```

Comment:
lets put at top since they're used thru rest of file. Or immediately before the first class to use them